src/passes/cairoUtilImporter.ts

```diff
@@ -41,6 +49,17 @@ export class CairoUtilImporter extends ASTMapper {
     }
   }
 
+  visitFunctionCall(node: FunctionCall, ast: AST): void {
+    // FIXME: remove when BitAnd is available for integer types in
+    // corelib
+    if (node.vFunctionName === CUTOFF_DOWNCAST[1]) {
+      const path = WARPLIB_INTEGER.slice(0, -1);
+      const name = WARPLIB_INTEGER.at(-1)!;
```

Comment:
The `!` can be misused, I would advise against it in every case even if it means writing a little more code. Even if it's safe here (and makes sense), I would prefer a different handling that wouldn't require it.

I believe there is a general risk of using `!` through out the code, JavaScript is unsafe enough for us to start adding the possibility of  human error during Typescript compilation

Reply:
I suggest allowing non-null operator for checks for internal consistencies. In other words, situations where we would otherwise write `assert (x !== null)`. It doesn't include handling business logic, business logic requires explicit handling and possibly throwing an appropriate exception.
We solve misuses with reviewing.

Reply:
We asked the team about their opinion, but no one answered, so we are keeping the restriction on non-null operators. I've added a commit removing them